docs/guides/integration-options-private-operator.md

```diff
@@ -0,0 +1,77 @@
+
+The participant must sign a contract (see [Account Setup](../getting-started/gs-account-setup.md)) to host a Private Operator instance.
+
+:::note
+A Private Operator has no visibility into the raw UID2s or UID2 tokens processed by a Public Operator or another Private Operator. Each Private Operator is a completely closed infrastructure.
+:::
+
+## Private Operator Workflow
+
+Every Private Operator runs in an enclave (AWS), confidential space (GCP), or confidential computing environment, (Azure). Each of these ensures that the Private Operator runs in a protected memory space.
+
+The basic workflow is as follows:
+
+1. On startup, the Private Operator goes through an attestation process with the Core service. The attestation process verifies that the Operator is running in a secure and memory-hardened environment that is known and trusted, and that the environment hasn't been tampered with.
+
+1. When the Operator passes that attestation process, the Core service gives the Private Operator a secure S3 URL for retrieving the information it needs for startup.
+
+1. The Private Operator retrieves the security information from Amazon S3 that it needs to process UID2s, such as salts, encryption keys, and user opt-out records. For security details, see [Private Operator Security](#private-operator-security).
+
+1. If an Operator is restarted, it goes through the attestation process again, and retrieves a fresh set of security information.
+
+1. The Operator re-attests periodically with Core to ensure that it is still running in a protected environment. If any compromise is detected, the Operator shuts down.
+
+The information exchange between the UID2 Core service and the UID2 Opt-Out service is secured using TLS. (**GWH_TM01: is this actually part of the Private Operator?**)
+
+## Private Operator Security
+
+Each supported Private Operator implementation must meet rigorous security standards. Some security points include:
+
+- The environment is secure and memory-hardened.
+- The Private Operator must complete an attestation process before accessing the information needed to process UID2s.
+- The information on S3 is encrypted at rest, and is encrypted in transit through TLS.
+- The information retrieved at startup is not stored locally at any point. It is only ever held in memory, and the Private Operator is running in a protected environment that makes it difficult for anyone running the Operator (an Administrator), as well as any external players, to see the data that's in memory.
+
+## Private Operator Integration Options
+
+The following Private Operator integrations are available.
+
+There is no functional difference between the Private Operator versions. Choose the integration type that fits best with your current cloud provider deployments.
```

Comment:
There is a change in voice here - suggest: Integrators are free to choose the integration type that fits best with their current cloud provider deployments.

Reply:
Good point. I removed that second sentence... LMK if you think we need it. Just feel it's slightly awkward, and if there's no functional difference it seems obvious that they can therefore choose the one that suits best.